administrator-manual/en/nextcloud.rst

```diff
@@ -33,6 +34,11 @@ After the installation a new application widget is added to the |product| web in
             Server logs keep track of which apps were disabled. After a successful update/upgrade procedure
             you can use the Applications page to update and re-enable the apps.
 
+.. note::   Nextcloud version 13 uses new PHP 7.1 (`nethserver-rh-php71-php-fpm`) while older version uses PHP 5.6 (`nethserver-rh-php56-php-fpm`).
+            If you have upgraded Nextcloud from a previous version, both php packages are installed. So you can remove php56 version (if it is
```

Comment:
You can remove php56 version (if there are no dependency problems) with the command "yum remove nethserver-rh-php56-php-fpm".